ocaml/libs/stunnel/stunnel.ml

```diff
@@ -157,6 +157,9 @@ let pool =
   ; cert_bundle_path= "/etc/stunnel/xapi-pool-ca-bundle.pem"
   }
 
+let remote_host remote_host_cert_file =
```

Comment:
Don't think the name is descriptive enough. Is this for a host outside the pool? Maybe `external_host`?